Services/data/devicemodels/scripts/EmergencyValveRelease-method.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+/*global log*/
+/*global updateState*/
+/*global sleep*/
+/*jslint node: true*/
+
+"use strict";
+
+/**
+ * Entry point function called by the simulation engine.
+ *
+ * @param context        The context contains current time, device model and id
+ * @param previousState  The device state since the last iteration
+ */
+
+/*jslint unparam: true*/
+function main(context, previousState) {
```

Comment:
The latest scripts include "previousProperties" as a parameter and have a more descriptive comment for the main() function. Should this also match that style?